apps/worker/src/app/workflow/services/workflow-worker.service.ts

```diff
@@ -31,8 +32,10 @@ interface IJobData {
   _userId: string;
 }
 
+const LOG_CONTEXT = 'WorkflowWorkerService';
+
 @Injectable()
-export class WorkflowQueueService extends QueueService<IJobData> {
+export class WorkflowWorkerService extends QueueService<IJobData> implements INovuWorker {
```

Comment:
Renaming as here we are instantiating a worker from BullMQ so for me the naming was confusing. Opinionated and debatable. Can roll back if needed.

Reply:
Renaming and refactor will be done in a different PR.